package.json

```diff
@@ -11,7 +11,7 @@
     "src/*"
   ],
   "peerDependencies": {
-    "typescript": "~3.9.5"
+    "typescript": "^4.0.0-dev"
```

Comment:
We should use patterns like `~4.0` so that we don't implicitly pull in e.g. 4.1.

Reply:
I went with ~4.0.0-dev so that we get the dev packages and patch levels in 4.0, but not 4.1